src/rebuild/srv.c

```diff
@@ -1298,8 +1251,14 @@ rebuild_task_ult(void *arg)
 		iv.riv_seconds          = rgt->rgt_status.rs_seconds;
 		iv.riv_stable_epoch	= rgt->rgt_stable_epoch;
 
-		rc = rebuild_iv_update(pool->sp_iv_ns, &iv, CRT_IV_SHORTCUT_NONE,
-				       CRT_IV_SYNC_LAZY, true);
+		if (!is_rebuild_global_done(rgt) || rgt->rgt_status.rs_errno != 0 ||
+	    	    task->dst_rebuild_op == RB_OP_REINT || task->dst_rebuild_op == RB_OP_EXTEND) {
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs